src/parsec_service/tests/README.md

```diff
@@ -16,9 +16,10 @@ multithreaded client. Valid requests are sent intermittently so as to check that
 still up and working correctly.
 
 The `ci.sh` script executes all tests and is used on the CI.
-[`rustfmt`](https://github.com/rust-lang/rustfmt) and
-[`clippy`](https://github.com/rust-lang/rust-clippy) are needed for code formatting and static
-checks.
+
+Parsec's code style is enforced by [`rustfmt`](https://github.com/rust-lang/rustfmt) and
+[`clippy`](https://github.com/rust-lang/rust-clippy), which are needed for code formatting and
```

Comment:
I think this needs to be changed, clippy is only for static lint checks.

Reply:
Done. I think that is clearer to what I meant :)